packages/paste-website/src/pages/foundations/illustrations/index.mdx

```diff
@@ -37,28 +39,143 @@ export const getStaticProps = async () => {
+</Box>
+
+<Callout variant="neutral" marginY="space70">
+  <CalloutText>
+    Consider the context before adding a photo illustration, as they often
+    represent specific user types. Use restraint, especially in busy or
+    heavily branded environments.
+  </CalloutText>
+</Callout>
+
+## Using Illustrations
+
+You can access a selection of illustrations for products available in the
+[Figma Web Spot Image library](https://www.figma.com/file/tr1bGht8drBOMaeX5EpJ7m/Web-Spot-Image-Repository?type=design&node-id=2310-5106&mode=design),
+which is managed by the Web and Brand teams. Turn it on in your local files for easy access, and use the
+illustrations in the “Product Illustrations” sticker sheet
+
+For the complete illustration collection, browse the [illustration library in Bynder](https://library.twilio.com/collections/view/ECEB801C-323E-4ED7-AADD4A152F5F6D92/) (VPN required).
+
+### Examples
+
+Illustrations are used sporadically in the product, and should support the
+overall message, not just be visual ornaments. These are some common uses of
+illustrations around the product.
+
+#### 1. Welcome and Get Started screens
+
+![](/images/illustrations-foundation/welcome-get-started.png)
+
+Illustrations are commonly placed on the right side of the header section to offer
+a friendly welcome to users. Keep each welcome screen to only 1 illustration to
+prevent overcrowding and maintain a minimalist aesthetic.
+
+
+#### 2. Empty states
+
+![](/images/illustrations-foundation/empty-state.png)
+
+Illustrations are positioned on the left side (or top for small screens) when users
+access a new/empty part of the experience. It helps to add visual interest, ensuring
+users don't miss what they need to do to continue.  Read more about the [empty state pattern](/patterns/empty-state).
+
+#### 3. Error states
+
+![](/images/illustrations-foundation/error-state.png)
+
+Illustrations are placed on the left side (or top for small screens) for error states,
+aimed to alleviate frustration. Keep in mind using multiple illustrations for different
+error states on the same screen may lead to confusion and increased frustration.
+Follow our [Error state pattern](/patterns/empty-state#error-states) carefully when adding error states and illustrations.
+
+#### 4. Highlights
+
+![](/images/illustrations-foundation/highlight.png)
+
+Illustrations are used to draw customers' attention to upsell and cross-sell opportunities.
+They serve as eye-catching elements, but it's important to be mindful of the number of
+illustrations on the screen. If every upsell or cross-sell opportunity is illustrated,
+it can diminish their effectiveness. Read our guidelines for
+[marketing Cards and cross-sell opportunities](/components/card#marketing-card).
+
+
+### Customization
+
+- If a pre-made illustration doesn’t fit the layout you can slightly adjust to
+better match your design.
+
+![](/images/illustrations-foundation/customization.png)
```

Comment:
**nitpick:**  Please update this image with the one attached here (trasnparent background) so it looks good on dark-mode.
[Illustration-customization.png.zip](https://github.com/twilio-labs/paste/files/14923445/Illustration-customization.png.zip)